cogs/socials.py

```diff
@@ -261,7 +261,7 @@ async def gpt(self, ctx: discord.ApplicationContext, text: str):
         await ctx.defer()
         messages.reverse()
         url = "https://free.churchless.tech/v1/chat/completions"
-        gpthistory = [{"role": "system", "content": f"{data.gaslight} The user's name is {ctx.author.display_name}. Do not use the user's full name, use their call name derived from their full name."}]
+        gpthistory = [{"role": "system", "content": f"{get_gaslight(ctx.guild.emojis, ctx.author.display_name)}"}]
```

Comment:
I doubt the code works on your machine because you never await the async get_gaslight function.